pycsob/client.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, merchant_id, base_url, private_key, csob_pub_key):
         """
         self.merchant_id = merchant_id
         self.base_url = base_url
-        self.key = self._get_key(private_key)
+        self.key = private_key
         self.pubkey = self._get_key(csob_pub_key)
```

Comment:
Isn't the same problem with the public key?

Reply:
Agreed, i would do this here as well.

Reply:
No it should not be a problem because we retrieve public key from file.

Reply:
I see, OK then.